api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -740,7 +751,249 @@ public void shouldTestPurgeProgramAttributeType() throws Exception {
                 int totalAttributeTypes = pws.getAllProgramAttributeTypes().size();
 		pws.purgeProgramAttributeType(programAttributeType);
 		assertEquals((totalAttributeTypes - 1), pws.getAllProgramAttributeTypes().size());
-}
+	}
+	
+	
+	@Test
+	public void getPrograms_shouldTestGetPrograms() {
+		List<Program> programs = pws.getPrograms("MAL");
+		List<Program> programs1 = pws.getPrograms("PR");
+		assertEquals(programs.size(), 1);
+		assertEquals(programs1.size(), 3);
+	}
+	
+	@Test
+	public void retireProgram_shouldTestRetireProgram() {
```

Comment:
`retireProgram_shouldTestRetireProgram` is not descriptive enough what it has to do

Reply:
I put there a comment

Reply:
```suggestion
	public void retireProgram_shouldSetRetiredStateToFalseAndSetAReason() {
```

I meant to change the test case name. Please remove the comment. It is unnecessary. Also, you forgot to assert the retired reason in your test case.